docs/features.rst

```diff
@@ -324,10 +321,10 @@ portability
 
 libtorrent runs on most major operating systems, including Windows,
 MacOS X, Linux, BSD and Solaris.
-It uses Boost.Thread, Boost.Filesystem, Boost.Date_time and various other
-boost libraries. At least version 1.46.1 of boost is required.
+It uses Boost.Thread, Boost.Asio, Boost.Chrono, Boost.Random, Boost.Date_time
```

Comment:
I think that for `master` you can remove Boost.Chrono, Boost.Random, Boost.Date_time

Reply:
yeah, I was tempted to do that in the merge, but I prefer to do it in a separate commit